README.md

```diff
@@ -1 +1,23 @@
 # hpqtypes-effectful
+
+## Description
+
+`effectful` bindings for the `hpqtypes` haskell bindings for `libpqtypes`
+
+## How to use
+
+This library exposes the `EffectDB` type-level effect that you can declare in your type signatures.
+
+An `Eff es` stack that contains `EffectDB` can use all functions
+that have a `MonadDB` constraint.
+
+example:
+```haskell
+exampleProgram :: Eff '[EffectDB, Error HPQTypesError, IOE] ()
```

Comment:
Not sure it makes sense to keep `Error HPQTypesError` in the example, since the example doesn't rely on it.  On the other hand, the only interpreter we currently provide does require the `Error HPQTypesError` constraint :thinking:.

Reply:
Removed